components/authentication-framework/org.wso2.carbon.identity.application.authentication.endpoint.util/src/main/java/org/wso2/carbon/identity/application/authentication/endpoint/util/EndpointConfigManager.java

```diff
@@ -50,6 +51,7 @@ public class EndpointConfigManager {
     private static char[] appPassword = null;
     private static String serverOrigin;
     private static boolean initialized = false;
+    private static String googleOneTapRestrictedBrowsers = "";
```

Comment:
Let's use `String.EMPTY` instead of `""`.

Reply:
Fixed 38d0c51fd441a7ee513b0b52d2459c3a732d9e2d